cdap-ui/app/features/tracker/templates/partial/add-preferred-tags-popover.html

```diff
@@ -0,0 +1,50 @@
+<!--
+  Copyright © 2016 Cask Data, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+<div class="modal-header">
+  <h3 class="modal-title">
+    <span>Add Preferred Tags</span>
+    <span class="fa fa-times pull-right" ng-click="$close()"></span>
+  </h3>
+</div>
+
+<div class="modal-body">
+  <div class="flex-box">
+    <div class="flex-item">
+      <div>Enter preferred tags here<br />
```

Comment:
This should be a textarea. The user should be able to type in the tags either comma separated or new line separated. We could still handle this in UI. May be in another PR but this needs to be a textarea according to mocks.